app/src/main/java/campus/tech/kakao/map/repository/Repository.kt

```diff
@@ -0,0 +1,36 @@
+package campus.tech.kakao.map.repository
+
+import android.util.Log
+import campus.tech.kakao.map.db.PlaceContract
+import campus.tech.kakao.map.db.PlaceDBHelper
+import campus.tech.kakao.map.model.Place
+
+class Repository (val dbHelper: PlaceDBHelper){
+    fun getAllPlace() : MutableList<Place>{
```

Comment:
getAllPlace 는 수정될 여지가 없기 때문에 MutableList<Place> 를 반환하기 보다는 Immutable 한 List<Place> 를 반환하는 것이 어떨까요?
```suggestion
    fun getAllPlace() : List<Place>{
```